models/barcelos2024resiliont/metadata.yaml

```diff
@@ -0,0 +1,34 @@
+title: Resilience Core Ontology
+acronym: ResiliOnt
+issued: 2024
+modified:
+contributor:
+ - https://dblp.org/pid/96/8280
+ - https://dblp.org/pid/63/9777
+ - https://dblp.org/pid/212/6684
+ - https://dblp.org/pid/134/4947
+ - https://dblp.org/pid/95/6356
+ - https://dblp.org/pid/72/6796
+ - https://dblp.org/pid/11/78
+keyword:
+ - resilience
+ - risk
+theme: Class H - Social Sciences
+editorialNote:
+ - Uploaded version 0.7.0
+ontologyType:
+ - core
+language: en
+designedForTask:
+ - conceptual clarification
+ - ontological analysis
+context:
+ - research
+source:
+ - https://raw.githubusercontent.com/pedropaulofb/resiliont/main/resources/Ontological%20Foundations%20of%20Resilience.pdf
```

Comment:
Replace this link with the DOI of the paper.

Reply:
There is no DOI yet. We can update the field once we have this piece of data.